ext/standard/string.c

```diff
@@ -2138,19 +2138,7 @@ static zend_string *php_chunk_split(const char *src, size_t srclen, const char *
 
 	chunks = srclen / chunklen;
 	restlen = srclen - chunks * chunklen; /* srclen % chunklen */
-
-	if (chunks > INT_MAX - 1) {
-		return NULL;
-	}
-	out_len = chunks + 1;
-	if (endlen !=0 && out_len > INT_MAX/endlen) {
-		return NULL;
-	}
-	out_len *= endlen;
-	if (out_len > INT_MAX - srclen - 1) {
-		return NULL;
-	}
-	out_len += srclen + 1;
+	out_len = (chunks + 1) * endlen + srclen +1;
```

Comment:
This isn't right as-is because the calculation may overflow. Possibly safe_alloc may be used.